.github/workflows/tests_lkcpu_python.yml

```diff
@@ -286,4 +297,4 @@ jobs:
         with:
           fail_ci_if_error: true
           verbose: true
-          token: ${{ secrets.CODECOV_TOKEN }}
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Do we want to add a blank line after to resolve the complain?

Reply:
I don't see a complaint anywhere. Let's merge it like that for now.